_pages/plugins/snt/faq.md

```diff
@@ -9,19 +9,17 @@ nav-title: FAQ
 
 ### How do I install SNT?
 
-See [Installation details](#SNT#Installation).
+See [Installation details](/plugins/snt/index#installation).
 
 ### How do I cite SNT?
 
-SNT is based on multiple publications. To cite SNT:
-
-  - Ferreira T, et al. (**2019**) (TBD)
+  - Arshadi, C., Günther, U., Eddison, M. Harrington, K.I.S, Ferreira T. [SNT: a unifying toolbox for quantification of neuronal anatomy](https://doi.org/10.1038/s41592-021-01105-7). *Nature Methods* 18, 374–377 (2021). 
```

Comment:
You can use the `{% include citation doi=... %}` syntax for this. Or even better, put this new DOI in the front matter of the `plugins/snt/index` page, and then you can write `{% include citation id='plugins/snt' %}`.

See https://imagej.net/editing/citations

Reply:
@tferr this sounds useful - is it something you want to do as part of this pr?